src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -281,13 +283,20 @@ def send_message(
         blocks=blocks,
     )
 
+    weblink = make_call(
+        client,
+        SlackAPIGetEndpoints.chat_permalink,
+        channel=response["channel"],
+        message_ts=response["ts"],
+    )
```

Comment:
I'd put this in a function you can call, so you don't have the same exact code in two places.

Reply:
Done.